### DISCUSSION TITLE
How to fead folder to neovide?

### DISCUSSION THREAD

**Discussion by eailfly** [neovide/neovide#1379] [Q&A]
neovide is working perfect if I start from terminal, if I start it from a git repo, the plugins like nvim-tree and project works good, telescope will only look for files under the repo as well.
My question is how should I feed directory or a git repo to neovide if it runs not from command line but let's say by double click the icon, the nvim-tree will show my entire home dir, and telescope is freeze because it need index all my home dir.
I think this is a common usage, after all the reason we use gui is we don't want terminal involved.
Thanks for any advise

**Comment by MultisampledNight** [comment]
Check out the `autochdir` option (see `:h autochdir`), which autonavigates the Neovim running in Neovide to the currently opened file. This way `neovide ~/wow/anywhere/config.toml` will end up with `~/wow/anywhere` as the current working directory.

**Reply by eailfly** [reply]
Hi MultisampledNight
thanks for the answer, but this still need open neovide from console, how about start it from launchpad like any other gui editors, vscode or even textedit, all gui editors have a function in menu called Open File or Open Folder(even Emacs has them too). I mean if we have to run neovide from console all the time, why bother to have a gui frontend?

**Reply by MultisampledNight** [reply]
Then you're looking for #1259, which isn't implemented (and not going to be happen from my side since I don't have nor use a macOS machine). The Automator script at https://github.com/neovide/neovide/issues/1259#issuecomment-1140337178 could be usable though.
